LibC/stdlib.cpp

```diff
@@ -329,60 +329,46 @@ size_t mbstowcs(wchar_t*, const char*, size_t)
     ASSERT_NOT_REACHED();
 }
 
-long strtol(const char* str, char** endptr, int base)
+long my_strtol(const char* str, char** endptr, int base)
 {
```

Comment:
my_strtol ?

Reply:
Lol whoops, that's not right! I had copied it out into a separate C file for testing.